qiskit/quantum_info/operators/measures.py

```diff
@@ -99,19 +105,21 @@ def process_fidelity(channel,
 
     # Validate complete-positivity and trace-preserving
     for label, chan in [('Input', channel), ('Target', target)]:
-        if isinstance(chan, Operator) and (require_cp or require_tp):
-            is_unitary = chan.is_unitary()
-            # Validate as unitary
-            if require_cp and not is_unitary:
-                raise QiskitError('{} channel is not completely-positive'.format(label))
-            if require_tp and not is_unitary:
-                raise QiskitError('{} channel is not trace-preserving'.format(label))
-        elif chan is not None:
-            # Validate as QuantumChannel
-            if require_cp and not chan.is_cp():
-                raise QiskitError('{} channel is not completely-positive'.format(label))
-            if require_tp and not chan.is_tp():
-                raise QiskitError('{} channel is not trace-preserving'.format(label))
+        if chan is not None and require_cp:
+            cp_cond = _cp_condition(chan)
+            neg = cp_cond < -ATOL_DEFAULT
```

Comment:
In the previous code the `is_cp` and `is_tp` use `self.atol` and `self.rtol` for tolerances. Why this is updated?